components/GeoAdd.coffee

```diff
@@ -0,0 +1,49 @@
+noflo = require 'noflo'
+
+# @runtime noflo-nodejs
+
+exports.getComponent = ->
+  c = new noflo.Component
+  c.description = 'Add a member to a geopositioned set'
+  c.icon = 'map-marker'
+  c.inPorts.add 'key',
+    datatype: 'string'
+  c.inPorts.add 'member',
+    datatype: 'string'
+  c.inPorts.add 'latitude',
```

Comment:
Isn't it very inconvenient to have separate ports for lat,lon(key,member) - don't these almost always come together?